### PR TITLE
more display clean up

### PR DESCRIPTION
**Pull request by nowickis**

### CODE REVIEW

File: resources/rooms/garden.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "features": ["shed", "bench"],
   "objects": [],
   "visited": false,
-  "longMSG": "You are in the garden. \n There are a gazebo and a shed.\n You may go...\n    south to the kitchen through the backdoor, or\n    southwest to the courtyard through the garden gate.\n",
```
Comment (nguyjere) [inline resources/rooms/garden.json]: Remove "gazebo" from messages.

File: resources/rooms/guestRoom.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "features": ["mirror","closet"],
   "objects": ["knife"],
   "visited": false,
-  "longMSG": "You are in the guestroom on the 2nd floor. \n There are a closet, a bed and a mirror.\n You may go...\n    north to the hallway 2 through the door\n",
```
Comment (nguyjere) [inline resources/rooms/guestRoom.json]: Remove bed from description

File: resources/rooms/secretRoom.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "features": ["gunSafe", "labTable", "metalCabinet"],
   "objects": ["gun", "distillingKit"],
   "visited": false,
-  "longMSG": "You are in the secret room on the 1st floor. \n There are a gunSafe and a lab table.\n You may go back to library by opening the sliding door.\n",
```
Comment (nguyjere) [inline resources/rooms/secretRoom.json]: add metal cabinet to description

File: resources/rooms/stewardRoom.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "features": ["desk","fileCabinet","zombieSteward"],
   "objects": [],
   "visited": false,
-  "longMSG": "You are in the steward room on the 2nd floor. \n There are a desk, a bed and a file cabinet...\nand......a Zombie Stewart smiling at you...no, showing his teeth does not mean smiling...\n You may go...\n    north to the hallway 2 through the door\n",
+  "longMSG": "You are in the steward room on the 2nd floor. \n There is a desk, a bed and a file cabinet...\nand......a Zombie Stewart smiling at you...no, showing his teeth does not mean smiling...\n You may go...\n    north to the hallway 2 through the door\n",
   "shortMSG": "You are in the steward room on the 2nd floor. \n Features: zombie stewart, bed, desk, fileCabinet \n Areas you may go to:\n    north - door - hallway 2 - 2nd floor\n",
```
Comment (nguyjere) [inline resources/rooms/stewardRoom.json]: Lets remove desk from features and description. And delete the json from the resource.

File: haunted_mansion_game/game_state.py [hunk 5]
```diff
@@ -77,6 +79,7 @@ def display_current_room(self):
         print "***ITEMS***"
         for item in current_room.objects:
             print self.get_item_by_name(item).displayName
+        '''
         print ""
```
Comment (nguyjere) [inline haunted_mansion_game/game_state.py]: Can you put this under an IF statement? And add function parameter "debug=false". So when we want to debug we just add "debug=true" to the function call.

File: resources/rooms/stewardRoom.json (hunk 4 above)
Comment (chiDuncans) [reply resources/rooms/stewardRoom.json]: I think we should keep nothing-happening features. We can discuss about it.

File: haunted_mansion_game/game_state.py (hunk 5 above)
Comment (chiDuncans) [reply haunted_mansion_game/game_state.py]: I will keep your recommendation note and will do make change as separated implementation commit.

**Pull request merged**